contrib/test_jenkins.sh

```diff
@@ -64,6 +64,8 @@ num_cpus=$(lscpu -p | grep -v '^#' | wc -l)
 [ -z $num_cpus ] && num_cpus=1
 parallel_jobs=4
 [ $parallel_jobs -gt $num_cpus ] && parallel_jobs=$num_cpus
+num_pinned_threads=$(taskset -pc $$ | awk -F': ' '{print $2}' | perl -lne 's/-/../g; @a=(eval); print scalar(@a)')
```

Comment:
maybe just do all parsing in perl?

Reply:
This version looks clener to me, WDYT?

Current:
```sh
... | awk -F': ' '{print $2}' | perl -lne 's/-/../g; @a=(eval); print scalar(@a)')
```
vs
```sh
... | perl -lne '$_=(split("list: "))[1]; s/-/../g; @a=(eval); print scalar(@a)')
```
(one char shorter)

Reply:
both seem equally unreadable to me :)

Reply:
`nproc` does the same, will update.